_data/downloads.yml

```diff
@@ -93,8 +93,8 @@ releases:
   Qubes release 4.0.4:
     testing: no
     latest: no
-    aging: yes
-    deprecated: no
+    aging: no
+    deprecated: yes
```

Comment:
Not true yet. See comment on this PR.

Reply:
I may wait til 4.0 depreciate

Reply:
> I may wait til 4.0 depreciate

Please do not submit PRs anticipating routine administrative/management actions. We will take care of it. If we somehow forget, than a PR would be appropriate.